packages/ai/cypress/specs/Input.cy.tsx

```diff
@@ -0,0 +1,950 @@
+        });
+    });
+
+    describe("Keyboard Shortcuts", () => {
+        it("should handle Shift+F4 to open menu", () => {
+            cy.mount(
+                <AIInput>
+                    <MenuItem slot="default" text="Generate"></MenuItem>
+                </AIInput>
+            );
+
+            cy.get("[ui5-ai-input]")
+                .shadow()
+                .find("input")
+                .focus()
+                .realPress(['Shift', 'F4']);
+
+            cy.get("[ui5-ai-input]")
+                .shadow()
+                .find("[ui5-menu]")
+                .ui5MenuOpened();
+        });
+
+        it("should handle Ctrl+Shift+Z for previous version when multiple versions exist", () => {
+            const onVersionChange = cy.spy().as("onVersionChange");
+
+            cy.mount(
+                <AIInput
+                    loading={false}
+                    currentVersion={2}
+                    totalVersions={3}
+                    onVersionChange={onVersionChange}
+                />
+            );
+
+            cy.get("[ui5-ai-input]")
+                .shadow()
+                .find("input")
+                .focus()
+                .realPress(['Control', 'Shift', 'Z']);
+
+            cy.get("@onVersionChange")
+                .should("have.been.calledOnce")
+                .its("firstCall.args.0.detail")
+                .should("deep.equal", {
+                    backwards: true
+                });
+        });
+
+        it("should handle Ctrl+Shift+Y for next version when multiple versions exist", () => {
+            const onVersionChange = cy.spy().as("onVersionChange");
+
+            cy.mount(
+                <AIInput
+                    loading={false}
+                    currentVersion={2}
+                    totalVersions={3}
+                    onVersionChange={onVersionChange}
+                />
+            );
+
+            cy.get("[ui5-ai-input]")
+                .shadow()
+                .find("input")
+                .focus()
+                .realPress(['Control', 'Shift', 'Y']);
+
+            cy.get("@onVersionChange")
+                .should("have.been.calledOnce")
+                .its("firstCall.args.0.detail")
+                .should("deep.equal", {
+                    backwards: false
+                });
+        });
+    });
+
+    describe("Input Integration", () => {
+        it("should inherit Input functionality", () => {
+            cy.mount(<AIInput value="Test content" />);
+
+            cy.get("[ui5-ai-input]")
+                .shadow()
+                .find("input")
+                .focus()
+                .should("have.value", "Test content")
+                .type(" additional text");
+
+            cy.get("[ui5-ai-input]")
+                .should("have.prop", "value")
+                .and("include", "Test content")
+                .and("include", "additional text");
+        });
+
+        it("should support readonly mode", () => {

```

Comment:
In readonly mode the AI icon is still interactable. Maybe in readonly state we should not display the AI icon at all or we can discuss with the team and designers about the expected behavior